example/spec/parallel/createReconciliationReport/CreateReconciliationReportSpec.js

```diff
@@ -311,6 +330,13 @@ describe('When there are granule differences and granule reconciliation is run',
         activeCollectionPromise,
       ]);
 
+      console.log('XXXXX Waiting for collections in list');
```

Comment:
Are we keeping this in?

Reply:
Yeah.  We leave it for debugging purpose